docs/resources.rst

```diff
@@ -0,0 +1,81 @@
+Resources
+=========
+
+Tutorials
+---------
+
+Django Debugging Tutorial
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Originally presented as an in-person workshop at DjangoCon US 2022, this
+tutorial by **Tim Schilling** covers debugging techniques in Django. Follow
+along independently using the slides and GitHub repository.
+
+* `Read the blog post <https://www.better-simple.com/django/2022/11/18/django-debugging-tutorial/>`__
+* `Follow along with the GitHub repo <https://github.com/tim-schilling/debug-tutorial/>`__
```

Comment:
```suggestion
* `Follow along with the GitHub repo <https://github.com/tim-schilling/debug-tutorial/>`__
* `View the slides <https://docs.google.com/presentation/d/1dmeFD5kGsukQaMinU0BJQyyCeu5ZGR4drc0pJilT10Y/edit?usp=share_link>`__
* `View the tutorial details <https://2022.djangocon.us/tutorials/it-doesnt-work-a-djangonauts-debugging/>`__
```

As much as I'd love to include my blog post in here, I don't feel like it contributes much for the reader here.